contrib/hbcurl/core.c

```diff
@@ -816,7 +822,18 @@ HB_FUNC( CURL_EASY_SETOPT )
             /* Error */
 
             /* HB_CURLOPT_ERRORBUFFER */
-            /* HB_CURLOPT_STDERR */
+             case HB_CURLOPT_STDERR:
+
+                if( hb_curl->pErr )
+                   fclose( hb_curl->pErr );
+
+                if( HB_ISCHAR( 3 ) )
+                {
+                   hb_curl->pErr = fopen( hb_parc( 3 ), "a"  );
```

Comment:
Extra space here.

Anyhow I'm not sure we need another place where FILE handles
are passed around. This is unsafe when using `libcurl` as a .dll.